src/components/SmartLabel.brs

```diff
@@ -147,13 +149,14 @@ function updateFonts(event = invalid)
   if event = invalid
     fontNames = m.fontFields
   else
-    fontNames = [event.getField()]
+    field = event.getField()
+    fontNames = [field.replace("Settings", "")]
   end if
 
```

Comment:
I'm not a fan of this change. We're iterating over **names** in the `fontNames` collection. Even if these _were_ styles, then the name would be `styleName`; not `style`.

Reply:
Updated